### DISCUSSION TITLE
My CS50 Codespace is frozen/blocked. How to restart/unfreeze

### DISCUSSION THREAD

**Discussion by elendiaar** [community/community#54457] [Codespaces]
### Select Topic Area

Bug

### Body

I was trying to submit a code to cs50 when the server returned a 500 error and asked me to restart the codespace. I tried that, nothing worked, tried refreshing the page, nothing worked. Now the codespace is shown as Active, but when I try to access it I keep getting the 500 error.

**Comment by yashoswalyo** [comment]
Codespace is down 
check: https://www.githubstatus.com/

**Reply by elendiaar** [reply]
Thanks, but it just allowed me to restart the Codespace, so it should be back online, hopefully. :)
